README.md

```diff
@@ -236,6 +236,18 @@ npm install mocha -g
 npm install --only=dev
 ```
 
+## Production Config
+- For production config, it should be added as a submodule.
+```sh
+git submodule add -b <branch> <https repository> config/env/production
+```
+- Whenever there are changes in production, you should update the submodule too.
+```sh
+git submodule init
+git submodule foreach git pull origin <branch>
```

Comment:
Please link @rohanfreedom 's doc

Also, I think the branch should be specified already in the submodule.